changelogs/unreleased/gh-5272-bootstrap-strategy-auto.md

```diff
@@ -0,0 +1,8 @@
+## feature/replication
+
+* Introduce new configuration option `bootstrap_strategy`. The default value of
```

Comment:
This needs to be marked as a breaking change.

Reply:
Marked them both.I guess changing the default value for `replication_sync_timeout` is a breaking change as well.
```diff
diff --git a/changelogs/unreleased/gh-5272-bootstrap-strategy-auto.md b/changelogs/unreleased/gh-527>
index 2496d352e..2614105be 100644
--- a/changelogs/unreleased/gh-5272-bootstrap-strategy-auto.md
+++ b/changelogs/unreleased/gh-5272-bootstrap-strategy-auto.md
@@ -1,8 +1,8 @@
 ## feature/replication
 
-* Introduce new configuration option `bootstrap_strategy`. The default value of
-  this option - "auto" - brings new replica behaviour on replica set bootstrap,
-  recovery and on replication reconfiguration. One of the notable changes is
-  that `replication_connect_quorum` option is not taken into account with
-  `bootstrap_strategy` = "auto". If you wish to return to the old behaviour, you
-  may use `bootstrap_strategy` = "legacy" (deprecated) (gh-5272).
+* **[Breaking change]** Introduce new configuration option `bootstrap_strategy`.
+  The default value of this option - "auto" - brings new replica behaviour on
+  replica set bootstrap, recovery and on replication reconfiguration. One of the
+  notable changes is that `replication_connect_quorum` option is not taken into
+  account with `bootstrap_strategy` = "auto". If you wish to return to the old
+  behaviour, you may use `bootstrap_strategy` = "legacy" (deprecated) (gh-5272).
diff --git a/changelogs/unreleased/gh-5272-replication-sync-timeout.md b/changelogs/unreleased/gh-52>
index 96ce907d1..e860b8e55 100644
--- a/changelogs/unreleased/gh-5272-replication-sync-timeout.md
+++ b/changelogs/unreleased/gh-5272-replication-sync-timeout.md
@@ -1,4 +1,4 @@
 ## feature/replication
 
-* Changed default value for `replication_sync_timeout` configuration option to
-  0 seconds (previous value was 300) (gh-5272).
+* **[Breaking change]]** Changed default value for `replication_sync_timeout`
+  configuration option to 0 seconds (previous value was 300) (gh-5272).
```